src/main/java/org/fundacion/jala/converter/core/PythonCompiler.java

```diff
@@ -83,4 +88,12 @@ public String makePythonFile(final String code) {
             return exception.getMessage();
         }
     }
+
+    /**
+     * Returns the result object for the operation.
```

Comment:
Add a line before the return, and also return and parameters has a dot at the end.

Reply:
Solved.